cmd/command/edge/image.go

```diff
@@ -246,3 +264,16 @@ func (p *Plural) writeCloudConfig(project, user, username, password, wifiSsid, w
 	_, err = file.WriteString(template)
 	return err
 }
+
+func (p *Plural) handleEdgeDownload(c *cli.Context) error {
+	outputDir := c.String("to")
+	url := c.String("url")
+
+	utils.Highlight("unpacking image contents\n")
+	if err := utils.Exec("docker", "run", "-i", "--rm", "--privileged", "-v", fmt.Sprintf("%s:/image", outputDir),
```

Comment:
does this still auth to the upstream registry with local credentials as expected?

Reply:
kk